app/src/test/java/com/futurice/freesound/functional/StringFunctionsTest.java

```diff
@@ -23,33 +23,33 @@
 public class StringFunctionsTest {
 
     @Test
-    public void isEmpty_returnsTrue_whenCalled_withEmptyString() {
-        assertThat(StringFunctions.isEmpty().call("")).isTrue();
+    public void isEmpty_returnsTrue_whenCalled_withEmptyString() throws Exception {
+        assertThat(StringFunctions.isEmpty().test("")).isTrue();
```

Comment:
Is`test` now the `call` for `Predicate`? I'd like to think that API will be renamed, it's a little confusing when compared with the use of `test` on `Observable`.